jenkins/ci.suse.de/templates/cloud-mkcloud-job-monasca-template.yaml

```diff
@@ -0,0 +1,28 @@
+- job-template:
+    name: 'cloud-mkcloud{version}-job-monasca-{arch}'
+    node: cloud-trigger
+    disabled: '{obj:disabled}'
+
+    triggers:
+      - timed: 'H H * * *'
+
+    logrotate:
+      numToKeep: -1
+      daysToKeep: 7
+
+    builders:
+      - trigger-builds:
+        - project: openstack-mkcloud
+          condition: SUCCESS
+          block: true
+          current-parameters: true
+          predefined-parameters: |
+            TESTHEAD=1
+            cloudsource=develcloud{version}
+            nodenumber=4
+            nodenumbercompute=3
```

Comment:
I was hoping to avoid adding another node type to the whole mkcloud machinery, but I guess we need to map it to controllers as it will likely require more memory..